src/test/scala/cromwell/binding/ThreeStepImportNamespaceSpec.scala

```diff
@@ -11,6 +11,9 @@ class ThreeStepImportNamespaceSpec extends FlatSpec with Matchers {
     |  output {
     |    File procs = stdout()
     |  }
```

Comment:
I don't disagree, I never found a place I really thought was a good one to handle the runtime (nor its own thing). However in this case the test is looking at a specific wdl and making sure everything imports correctly, so I don't see it as being a big deal either way.

That said, there should be wdl without runtime and verifying that it's empty.